lib/service.js

```diff
@@ -147,7 +148,9 @@ function processService(serviceMethod, env) {
                     }
                     resolve(result);
                 }
-                const result = verify(cfg, legacyCallback);
+                var callScope = new EventEmitter();
+                callScope.logger = new ComponentLogger({});
```

Comment:
are you sure we need to pass empty object?

Reply:
To awoid NPE?

Reply:
```
Logger.prototype.child = function (options, simple) {
    return new (this.constructor)(this, options || {}, simple);
}
```
you can skip options argument

Reply:
Ok, changed it